src/ContinuousEnergyNeutron.tex

```diff
@@ -961,6 +975,8 @@ \subsubsubsection{\var{LAW}=66---$N$-body phase space distribution}\label{sec:LA
   \label{eq:LAW66}
 \end{align}
 
+While MCNP will only use the values for \var{NPSX} and $A_{P}$ (given in \var{LDAT}(1) and \var{LDAT}(2)), NJOY does add the derived probability density function and cumulative density function for $T(\xi)$ starting at \var{LDAT}(3).
```

Comment:
👍🏻

Reply:
As far as I know. Even the MCNP5 manual ACE format appendix only gives NPSX and AP as the ACE LAW = 66 XSS array and does not mention the xi part of the law. The NJOY source code does however write all these values to the ACe file.